lib/internal/fs/sync_write_stream.js

```diff
@@ -10,11 +10,11 @@ const { closeSync, writeSync } = require('fs');
 function SyncWriteStream(fd, options) {
   Writable.call(this, { autoDestroy: true });
 
-  options = options || {};
+  const { autoClose } = { ...options };
```

Comment:
```suggestion
  const { autoClose } = options;
```
I think this is better cause we can remove one `||` operator.

Reply:
sorry, I omitted the `undefined`

Reply:
Yeah, converting this to destructuring should work. :+1: